From 94d43f912264ce231be9072cf3899d29934da376 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Thu, 14 Jan 2021 14:09:06 +0700
Subject: [PATCH 365/442] ARM: dts: r8a7742: Remove unused VSP nodes

VSP modules vsp@fe920000, vsp@fe928000, vsp@fe930000, and vsp@fe938000
are used as VSP for DU and already defined in other nodes (vspm, vspd0,
and vspd1).

This patch removes them to avoid confliction.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742.dtsi | 55 ------------------------------------------
 1 file changed, 55 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index b467a9e..b86eb373 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -2034,61 +2034,6 @@
 			status = "disabled";
 		};
 
-		vsp@fe920000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe920000 0 0x8000>;
-			interrupts = <GIC_SPI 266 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7742_CLK_VSP1_R>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-sru;
-			renesas,#rpf = <5>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <4>;
-		};
-
-		vsp@fe928000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe928000 0 0x8000>;
-			interrupts = <GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7742_CLK_VSP1_S>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lut;
-			renesas,has-sru;
-			renesas,#rpf = <5>;
-			renesas,#uds = <3>;
-			renesas,#wpf = <4>;
-		};
-
-		vsp@fe930000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe930000 0 0x8000>;
-			interrupts = <GIC_SPI 246 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7742_CLK_VSP1_DU0>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lif;
-			renesas,has-lut;
-			renesas,#rpf = <4>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <4>;
-		};
-
-		vsp@fe938000 {
-			compatible = "renesas,vsp1";
-			reg = <0 0xfe938000 0 0x8000>;
-			interrupts = <GIC_SPI 247 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&mstp1_clks R8A7742_CLK_VSP1_DU1>;
-			power-domains = <&cpg_clocks>;
-
-			renesas,has-lif;
-			renesas,has-lut;
-			renesas,#rpf = <4>;
-			renesas,#uds = <1>;
-			renesas,#wpf = <4>;
-		};
-
 		du: display@feb00000 {
 			compatible = "renesas,du-r8a7742";
 			reg = <0 0xfeb00000 0 0x70000>,
-- 
2.7.4

